Userland/Libraries/LibJS/Print.cpp

```diff
@@ -137,29 +137,45 @@ ErrorOr<void> print_array(JS::PrintContext& print_context, JS::Array const& arra
 
 ErrorOr<void> print_object(JS::PrintContext& print_context, JS::Object const& object, HashTable<JS::Object*>& seen_objects)
 {
-    TRY(js_out(print_context, "{{"));
+    TRY(js_out(print_context, "{}{{", object.class_name()));
     bool first = true;
-    for (auto& entry : object.indexed_properties()) {
-        TRY(print_separator(print_context, first));
-        TRY(js_out(print_context, "\"\033[33;1m{}\033[0m\": ", entry.index()));
-        auto value_or_error = object.get(entry.index());
-        // The V8 repl doesn't throw an exception here, and instead just
+    static constexpr size_t max_number_of_new_objects = 20; // Arbitrary limit
+    size_t original_num_seen_objects = seen_objects.size();
```

Comment:
This is an ugly hack, but without it printing even the simplest HTML Element will end up printing essentially every single element in the DOM due to all the cross-links b/w nodes.